app/models/concerns/media.rb

```diff
@@ -1,6 +1,8 @@
 module Media
   extend ActiveSupport::Concern
 
+  STATUSES = %w[nya upcoming current finished].freeze
```

Comment:
I think `not_yet_airing` would be nicer, there's not a lot of reason to save those like, 10 bytes